cristin-commons/src/test/java/no/unit/nva/utils/CognitoUtilCreateUsers.java

```diff
@@ -0,0 +1,36 @@
+package no.unit.nva.utils;
+
+import no.unit.nva.cognito.CognitoUtil;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.util.Objects.isNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@Tag("integrationTest")
+@Tag("createTestUsers")
+public class CognitoUtilCreateUsers {
+
+    public static final String FEIDE_ID = "karate-user-administrator@sikt.no";
+    public static final String PASSWORD = "p@ssW0rd";
+    private static final String USER_POOL_ID = "eu-west-1_DNRmDPtxY";
+    private static final String CLIENT_APP_ID = "4qfhv3kl9qcr2knsfb8lhu1u40";
+
+    private static final Logger logger = LoggerFactory.getLogger(CognitoUtilCreateUsers.class);
+
+
+    @Test
+    void createTestUser() {
+        if (isNull(CognitoUtil.loginUser(FEIDE_ID, PASSWORD, USER_POOL_ID, CLIENT_APP_ID))) {
+            logger.info("createTestUser");
+            CognitoUtil.deleteUser(FEIDE_ID, USER_POOL_ID);
```

Comment:
You delete even if `isNull(CognitoUtil.loginUser...` ?

Reply:
Just recreating the the user if login don't return a token - login could fail if password is wrong, delete doesn't care if user exist or not.